File: 1281/solution.py
class Solution(object):
    def subtractProductAndSum(self, n):
        """
        :type n: int
        :rtype: int
        """
        str_n = str(n)
        product = 1
        sum = 0
        for i in str_n:
            x = int(i)
            product *= x
            sum += x

        return product - sum

File: 1281/solution.cs
public class Solution {
    public int SubtractProductAndSum(int n) {
        string str = n.ToString();
        int product = 1;
        int sum = 0;
        for(int i = 0; i < str.Length; i++){
            int x = str[i] - '0';
            product *= x;
            sum += x;
        }

        return product - sum;
    }
}
